server/src/components/isShip.ts

```diff
@@ -16,6 +16,12 @@ export class IsShipComponent extends Component {
    */
   category: string = "Cruiser";
 
+  /**
+   * The current alert level of the ship. On a scale from 5 being "all clear" and 1 being "red alert".
+   * p represents a cloaked status.
+   */
+  alertLevel: string = "5";
```

Comment:
It might be useful to create an `AlertLevel` type that is a union of all the possible alert levels, like 

```ts
type AlertLevel = "5" | "4" | "3" | "2" | "1" | "p";
```

That enumerates all the available values, and makes it so you can't easily set the alert level to an invalid value. That type could be reused elsewhere as needed.